### DISCUSSION TITLE
The document file cant upload to docker's /data/doc

### DISCUSSION THREAD

**Discussion by garyman123** [open-webui/open-webui#3633] [General]
# Bug Report

I upload a txt file in Documents,use #+filename in prompt ,but model dosnt red the txt. Then I cheak the path, there is nothing in the docker's volums /data/doc. The RAG can work and the Vector Storage is not empty

**Bug Summary:**

**Steps to Reproduce:**

**Expected Behavior:**

**Actual Behavior:**

## Environment

- **Open WebUI Version:** last
- **Ollama (if applicable):** last

- **Operating System:** windows 2016
- **Browser (if applicable):** edge

## Reproduction Details

**Confirmation:**

- [ ] I have read and followed all the instructions provided in the README.md.
- [ ] I am on the latest version of both Open WebUI and Ollama.
- [ ] I have included the browser console logs.
- [ ] I have included the Docker container logs.

## Logs and Screenshots

**Browser Console Logs:**
[Include relevant browser console logs, if applicable]

**Docker Container Logs:**
[Include relevant Docker container logs, if applicable]

**Screenshots (if applicable):**
![image](https://github.com/open-webui/open-webui/assets/134024904/b0d43b61-67ea-4bd1-b5c0-d31002519f23)
![image](https://github.com/open-webui/open-webui/assets/134024904/cdad0da2-3ac5-46b6-8288-3c1b17e8a708)
![image](https://github.com/open-webui/open-webui/assets/134024904/d5aed7a2-f1de-425e-91b7-c684cd15f684)
![image](https://github.com/open-webui/open-webui/assets/134024904/4c6263b6-373b-4775-a125-7665af80c811)
![image](https://github.com/open-webui/open-webui/assets/134024904/9a5a4135-c946-4465-b7a2-d43a4071c337)

## Installation Method

[Describe the method you used to install the project, e.g., manual installation, Docker, package manager, etc.]

## Additional Information

[Include any additional details that may help in understanding and reproducing the issue. This could include specific configurations, error messages, or anything else relevant to the bug.]

## Note

If the bug report is incomplete or does not follow the provided instructions, it may not be addressed. Please ensure that you have followed the steps outlined in the README.md and troubleshooting.md documents, and provide all necessary information for us to reproduce and address the issue. Thank you!

**Comment by garyman123** [comment]
![Snipaste_2024-07-05_10-09-15](https://github.com/open-webui/open-webui/assets/134024904/68511e32-bfc2-4411-8f1f-1f558683cf34)
The embedding can vectorize the document.But llm cant answers what the document about . I  don't know if it's because  the document file not in data/docs, I see the "Scan for documents from DOCS_DIR (/data/docs)" in the admin setting

**Reply by rogeryangugdsb** [reply]
You may check the function store_doc in main.py @ https://github.dev/open-webui/open-webui to see where the files are stored.
